### DISCUSSION TITLE
lessons from FarPlay?

### DISCUSSION THREAD

**Discussion by mawess** [jamulussoftware/jamulus#3219] [General]
FarPlay seems to make more noise about itself.
has anybody tested and made comparing measurements?

**Comment by dcorson-ticino-com** [comment]
Not really interesting as the terms of use state it is intended only for people in the United States.

**Reply by mawess** [reply]
i mean as an inspiration to make jamulus better... not as an alternative.

**Reply by mcfnord** [reply]
I think in terms of niches. In the 2-participant lossless recording category, they might be the best. I haven't tried all the apps similar to Jamulus. I bet most are the best for at least one scenario.

Here's an audio-video session between two people using FarPlay:
https://www.youtube.com/watch?v=aRIQNHcXyyQ

**Comment by pcar75** [comment]
Some tidbits ...
Seems like a do-it-all (incl. video) miracle ... but lots of marketing blurb.
Most testimonies between 2 participants.
Peer-to-peer architecture, so probably affected by the number of participants and obviously the capacity and quality of their internet connections.
Seems based on netjack2 and Jacktrip
Transmits uncompressed full-duplex 48k/16b PCM when you record.
Recording separate tracks for later DAW mixing
You need to register and login to download
45 min. one-on-one sessions are free (w/wo video ?)

Ref.: https://www.soundonsound.com/forum/viewtopic.php?t=82691

Just my 2¢  ;-)

**Comment by softins** [comment]
Martin Waugh in Scotland has been trying it out and has posted a few videos on Youtube about it. He is a long-time member of the WorldJam community.

See https://www.youtube.com/@jamjazz/videos
